### ISSUE TITLE
Origin missing from mysql-metrics since 8.20.0

### ISSUE THREAD

**Issue by youngm** [cloudfoundry/mysql-monitoring-release#4]
We recently upgraded from 8.20.0 to 9.2.0 and noticed that all the metrics emitted by mysql-metrics now have an empty "origin" attribute.  Perhaps something broke in the migration to loggregator v2?  Other v2 metrics have an origin.  Only p-mysql metrics are missing the origin.

To duplicate deploy latest mysql-metrics job and examine the firehose.  Notice that all components have an origin except for p-mysql metrics.

**Comment by youngm** [comment]
Sample config:
```
  - name: mysql-metrics
    release: mysql-monitoring
    properties:
      mysql-metrics:
        host: localhost
        password: -- redacted --
        metrics_frequency: 120
        log_metrics_to_disk: false
        tls:
          ca: -- redacted --
          cert: -- redacted --
          key: -- redacted --
```
Example output:
```
origin:"" eventType:ValueMetric timestamp:1542302793619368036 deployment:"cf1-dev" job:"data" index:"bc1e68d2-f892-482d-af53-3c642d0d6958" ip:"10.118.216.194" tags:<key:"source_id" value:"p-mysql" > valueMetric:<name:"/p-mysql/galera/wsrep_cluster_status" value:1 unit:"number" >
```

**Comment by abg** [comment]
I am not sure why this wasn't updated and closed out when the team fixed this in 2019 in 8c37e60b5cfc4b9ee1c440b736a39a9addd6506d (9.20.0+) and delivered the story linked in the previous comment.

I am going to close this issue now.
